src/ontology/cl-edit.owl

```diff
@@ -31877,6 +31879,15 @@ AnnotationAssertion(<http://purl.org/dc/terms/date> obo:CL_4033054 "2023-08-22T1
 AnnotationAssertion(rdfs:label obo:CL_4033054 "perivascular cell")
 EquivalentClasses(obo:CL_4033054 ObjectIntersectionOf(obo:CL_0000003 ObjectSomeValuesFrom(obo:RO_0002220 obo:UBERON_0002049)))
 
+# Class: obo:CL_4033055 (airway submucosal gland duct ciliated cell)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:17707699") Annotation(oboInOwl:hasDbXref "PMID:30864819") obo:IAO_0000115 obo:CL_4033055 "A ciliated cell that is part of a ciliated duct of an airway submucosal gland.")
+AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_4033055 <https://orcid.org/0000-0001-6677-8489>)
+AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_4033055 <https://orcid.org/0000-0002-9185-3994>)
+AnnotationAssertion(<http://purl.org/dc/terms/date> obo:CL_4033055 "2023-09-08T10:32:27Z"^^xsd:dateTime)
+AnnotationAssertion(rdfs:label obo:CL_4033055 "airway submucosal gland duct ciliated cell")
+EquivalentClasses(obo:CL_4033055 ObjectIntersectionOf(obo:CL_0000064 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_8600014)))
```

Comment:
"airway" is noted in label and text definition, but this association is missing from the logical definition.